pkg/compactor/block_visit_marker.go

```diff
@@ -18,24 +18,26 @@ import (
 	"github.com/thanos-io/thanos/pkg/block/metadata"
 )
 
-const BlockVisitMarkerFile = "block.visit"
```

Comment:
Not sure why it is named differently at the beginning, especially the suffix `visit`. Does it intend to be named differently from the Thanos markers?

Reply:
I agree.. we should keep in the same pattern..

Maybe `visit-mark.json`?

Reply:
Yeah updated the name.